### PR TITLE
sci-libs/clblast: Fixed error when building with CUDA enabled.

### PR DESCRIPTION
**Pull request by pyeatt**
Signed-off-by: Larry D. Pyeatt <larry.pyeatt@sdsmt.edu>

### CODE REVIEW

File: sci-libs/clblast/files/patches/level2_xtrsv [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- CLBlast/src/routines/level2/xtrsv.cpp	2022-05-21 08:44:13.811816246 -0600
```
Comment (thesamesam) [inline sci-libs/clblast/files/patches/level2_xtrsv:1]: No need to put patches in a patches/ subdir.

Could you move them into just `files/`, and use a `.patch` filename suffix?

File: sci-libs/clblast/files/patches/level3_xtrsv [hunk 2]
```diff
@@ -0,0 +1,11 @@
+--- CLBlast/src/routines/level3/xtrsm.cpp	2022-05-21 08:44:13.815816361 -0600
++++ CLBlast_fixed/src/routines/level3/xtrsm.cpp	2022-05-21 08:28:06.756355738 -0600
```
Comment (thesamesam) [inline sci-libs/clblast/files/patches/level3_xtrsv:2]: Could you put a comment at the top of each of the patches explaining what it's for (and ideally why it works/how)? Also include any links to upstream submission/bugs?

File: sci-libs/clblast/files/patches/level2_xtrsv (hunk 1 above)
Comment (pyeatt) [reply sci-libs/clblast/files/patches/level2_xtrsv]: Testing those changes now.

Comment (pyeatt) [reply sci-libs/clblast/files/patches/level2_xtrsv]: Done. Pushed new commit.

File: sci-libs/clblast/files/patches/level3_xtrsv (hunk 2 above)
Comment (pyeatt) [reply sci-libs/clblast/files/patches/level3_xtrsv]: Done.  Pushed new commit.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**